demo/starter/slides-feat.md

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
The starter is meant to be an example showcase and a playground. For tests like this it's better to move to test fixtures.

Reply:
Sorry for accidentally deleting the branch, fix that & I have [opened new PR](https://github.com/slidevjs/slidev/pull/862) so we can continue from there.